stories/FileTree.stories.tsx

```diff
@@ -1,72 +1,85 @@
-        mime_category: 'directory',
-      },
-      'background-scripts/index.js': {
-        ...fakeVersionEntry,
-        depth: 1,
-        filename: 'index.js',
-        path: 'background-scripts/index.js',
-      },
-      'background-scripts/libs': {
-        ...fakeVersionEntry,
-        depth: 1,
-        filename: 'libs',
-        mime_category: 'directory',
-        path: 'background-scripts/libs',
-      },
-      'background-scripts/libs/jquery.min.js': {
-        ...fakeVersionEntry,
-        depth: 2,
-        filename: 'jquery.min.js',
-        path: 'background-scripts/libs/jquery.min.js',
-      },
-      'background-scripts/extra': {
-        ...fakeVersionEntry,
-        depth: 1,
-        filename: 'extra',
-        mime_category: 'directory',
-        path: 'background-scripts/extra',
-      },
-      'styles.css': {
-        ...fakeVersionEntry,
-        filename: 'styles.css',
-        path: 'styles.css',
+const getVersion = () => {
+  return createInternalVersion({
+    ...fakeVersion,
+    file: {
+      ...fakeVersion.file,
+      entries: {
+        'manifest.json': {
+          ...fakeVersionEntry,
+          filename: 'manifest.json',
+          path: 'manifest.json',
+        },
+        'background-scripts': {
+          ...fakeVersionEntry,
+          filename: 'background-scripts',
+          path: 'background-scripts',
+          mime_category: 'directory',
+        },
+        'background-scripts/index.js': {
+          ...fakeVersionEntry,
+          depth: 1,
+          filename: 'index.js',
+          path: 'background-scripts/index.js',
+        },
+        'background-scripts/libs': {
+          ...fakeVersionEntry,
+          depth: 1,
+          filename: 'libs',
+          mime_category: 'directory',
+          path: 'background-scripts/libs',
+        },
+        'background-scripts/libs/jquery.min.js': {
+          ...fakeVersionEntry,
+          depth: 2,
+          filename: 'jquery.min.js',
+          path: 'background-scripts/libs/jquery.min.js',
+        },
+        'background-scripts/extra': {
+          ...fakeVersionEntry,
+          depth: 1,
+          filename: 'extra',
+          mime_category: 'directory',
+          path: 'background-scripts/extra',
+        },
+        'styles.css': {
+          ...fakeVersionEntry,
+          filename: 'styles.css',
+          path: 'styles.css',
+        },
       },
     },
-  },
-});
+  });
+};
 
 // We display an alert when a file has been selected.
 const onSelectFile = (path: string) => {
   // eslint-disable-next-line no-alert
   alert(`Selected file: ${path}`);
 };
 
-storiesOf('FileTree', module).add('default', () => (
-  <FileTree
-    linterMessages={undefined}
-    onSelect={onSelectFile}
-    version={version}
-  />
-));
+const render = ({
+  store = configureStore(),
+  ...moreProps
+}: { store?: Store } & Partial<FileTreeProps> = {}) => {
+  const props: FileTreeProps = {
+    onSelect: onSelectFile,
+    version: getVersion(),
```

Comment:
It just made me feel twitchy because code execution in modules leads to slow startup times as the app grows. It's hard to track down this type of slowness.

Reply:
I'm not sure I follow. This is in a storybook story. Do you mean slow startup for the storybook?

Reply:
mmh, I am confused here too.